### PR TITLE
Changes to Mini Village map

### PR DESCRIPTION
**Pull request by AnyBlue**
Added mine, removed some houses, made the glass in village houses breakable, added bronze sphere, added hint, fixed chairs in a base house. I don't really remember what we added yet but I think it's everything what we made.

### CODE REVIEW

File: mini_village/map.conf [hunk 1]
```diff
@@ -11,3 +11,4 @@ name = Mini Village
 initial_stuff = default:pick_steel,default:torch 5
 r = 115
 license = CC BY-SA 4.0
+hint = You can find treasures in the mine.
```
Comment (JostP) [inline mini_village/map.conf:14]:
```suggestion
```

Comment (JostP) [reply mini_village/map.conf:14]: Hint is not needed since there is no mese or diamonds in mine.

Comment (JostP) [reply mini_village/map.conf:14]: once you commit this, I will merge your map

**Comment by JostP** [comment]
Nice update. Maybe you removed too many houses, but let's test how will players like the update.

**Review by JostP — APPROVED** [review]

**Comment by Panquesito7** [comment]
> removed too many houses

Probably. I think they should add some back.

**Pull request merged**